### ISSUE TITLE
Add option to ignore trailing bytes

### ISSUE THREAD

**Issue by DerGuteWolf** [creationix/msgpack-js-browser#4]
It would be nice if there would be a option to simply ignore superfluous trailing bytes.

**Comment by creationix** [comment]
Agreed, or somehow report how many bytes are leftover or how many were consumed so that external users could decide to throw a fatal error or ignore.
